### PR TITLE
Work-in-progress of graphviz.go test

### PR DESCRIPTION
**Pull request by adamsigal**
## Tips:

* please read the style guide before submitting your patch:
[docs/style-guide.md](../docs/style-guide.md)

* commit message titles must be in the form:

```topic: Capitalized message with no trailing period```

or:

```topic, topic2: Capitalized message with no trailing period```

* golang code must be formatted according to the standard, please run:

```
make gofmt		# formats the entire project correctly
```

or format a single golang file correctly:

```
gofmt -w yourcode.go
```

* please rebase your patch against current git master:

```
git checkout master
git pull origin master
git checkout your-feature
git rebase master
git push your-remote your-feature
hub pull-request	# or submit with the github web ui
```

* after a patch review, please ping @purpleidea so we know to re-review:

```
# make changes based on reviews...
git add -p		# add new changes
git commit --amend	# combine with existing commit
git push your-remote your-feature -f
# now ping @purpleidea in the github PR since it doesn't notify us automatically
```

## Thanks for contributing to mgmt and welcome to the team!

### CODE REVIEW

File: pgraph/graphviz_test.go [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    G.AddEdge(v3, v1, e3)
+
+    G.AddEdge(v4, v5, e4)
+    G.AddEdge(v5, v6, e5)
+	G.AddEdge(v6, v4, e6)
+
+	var expectedResult = [...]string{
+        "digraph "g2" {",
+        "	label=\"g2\";",
+        "	\"0xc00004a510\" [label=\"v1\"];",
```
Comment (purpleidea) [inline pgraph/graphviz_test.go]: Quiz: What are these 0xc... things?

Comment (adamsigal) [reply pgraph/graphviz_test.go]: Data addresses.

Comment (purpleidea) [reply pgraph/graphviz_test.go]: Why would it be bad to have these in a test?

Comment (adamsigal) [reply pgraph/graphviz_test.go]: Because they change from machine to machine

**Pull request closed without merging**